modules/e2eping/e2eping.js

```diff
@@ -104,45 +107,35 @@ class ParticipantWrapper {
     }
 
     /**
-     * Goes over the requests, clearing ones which we don't need anymore, and
-     * if it finds at least one request with a valid RTT in the last
-     * 'analyticsIntervalMs' then sends an analytics event.
+     * Check if we've received the pre-configured number of responses, and if
+     * so log the measured RTT and stop sending requests.
      * @type {*}
      */
-    maybeSendAnalytics() {
+    maybeLogRttAndStop() {
         const now = window.performance.now();
 
-        // The RTT we'll report is the minimum RTT measured in the last
-        // analyticsInterval
+        // The RTT we'll report is the minimum RTT measured
         let rtt = Infinity;
         let request, requestId;
+        let numRequestsWithResponses = 0
 
-        // It's time to send analytics. Clean up all requests and find the
         for (requestId in this.requests) {
             if (this.requests.hasOwnProperty(requestId)) {
                 request = this.requests[requestId];
 
                 if (request.rtt) {
+                    numRequestsWithResponses++;
                     rtt = Math.min(rtt, request.rtt);
                 }
             }
         }
 
-        if (rtt < Infinity) {
-            this.sendAnalytics(rtt);
+        if (numRequestsWithResponses >= e2eping.numRequests) {
+            console.log(`Measured RTT=${rtt} ms to participant ${this.id} (in
+                region ${this.participant.getProperty('region')}`));
```

Comment:
This is the remote participant's connected region, rather than the region of the bridge they're connected to?

Reply:
Correct. How to log this data to rtcstats is still TBD

Reply:
@nils-ohlmeier or @andrei-gavrilescu can correct me if I'm wrong, but I think this either needs to be bubbled up to jitsi-meet (and add a method similar to [this](https://github.com/jitsi/jitsi-meet/blob/85c505a29d5b6d39cabb26c26a1fb054b3ce098e/react/features/rtcstats/RTCStats.js#L94) one to send the rtt back to the rtcstats server) or add the capability of sending stats entries from lib-jitsi-meet. Not sure which approach is better suited here. /cc @jallamsetty1 @hristoterezov in case you guys have any thoughts to share